weather-app/index.js

```diff
@@ -20,30 +21,75 @@ function renderTableRow() {
             <td>${temp}</td>
         </tr>
         `
+        cMap += `<img src='https://maps.locationiq.com/v2/staticmap?key=pk.c6c8e0f26d559c4194d567e972e29b74&center=${lat},${lon}&zoom=12&size=400x400&markers=${lat},${lon}|icon:large-red-cutout'></img>`
     })
     tableBody.innerHTML = tBody;
+    cityMap.innerHTML = cMap;
 }
 
 function getWeather(city){
     return fetch(`http://api.weatherstack.com/current?access_key=8a2380626129e4b4b0b74c8703d3d9c3&query=${city}`)
     .then(response => response.json())
+    .then((response) => {
+        const{
+            current: {temperature},
+            location: {country, name, lat, lon},
+        } = response;
+        map.set(name, [name, country, temperature, lat, lon])
+       renderTableRow();
+       localStorage.setItem(response.location.name, JSON.stringify(response));
+    });    
 }
 
 form.addEventListener('submit', function(event){
     event.preventDefault();
+    getWeather(input.value);
+})
 
-    getWeather(input.value).then((response) => {
-        const{
-            current: {temperature},
-            location: {country, name},
+// =========================== My Weather=================================
 
-        } = response;
-        map.set(name, [name, country, temperature])
-        renderTableRow();
+function getMyWeather({coords: {latitude: lat, longitude: lon}}){
+    console.log(`${lat}, ${lon}`)
+    fetch(`https://eu1.locationiq.com/v1/reverse.php?key=pk.80f74496452169d08d8ab400676ea8d9&lat=${lat}&lon=${lon}&format=json&accept-language=en`)
```

Comment:
You don't need to use  `https://eu1.locationiq.com` to get weather by location.

simply :

![image](https://user-images.githubusercontent.com/23724142/96775716-99e60700-13f0-11eb-8a00-4d1f2b429d41.png)